disk.py

```diff
@@ -23,11 +23,15 @@ def ensure_datadir():
     if not os.path.isdir(DATADIR):
         # make datadir
         os.mkdir(DATADIR)
+
+    wallet_dir = full_path('wallets')
```

Comment:
I make every check separately to avoid situation where `~/.junction` exists but `~/.junction/wallets` or `~/.junction/settings.json` doesn't ...